docs/getting_started.md

```diff
@@ -99,7 +99,7 @@ ever stuck, you can post a question in the
    `homeserver.yaml`):
    
    ```yaml
-    app_service_config_files:
```

Comment:
Unneeded?

Reply:
I don't remember changing this.
It might have been an auto formatter (which I turned off now), being bothered by the additional space character.

Reply:
Ah, `git blame` shows this as coming from `a0bc9448`.

If you like, you can put the space back in (and maybe squash it with fdddcf7) so that we have 0 diffs against `develop` other than changelogs.